src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureBackupClientAdapter/AzureBackupClientAdapter.cs

```diff
@@ -49,12 +34,6 @@ public partial class AzureBackupClientAdapter
         private BackupServicesManagementClient azureBackupClient;
 
         /// <summary>
-        /// Cancellation Token Source
-        /// </summary>
-        private CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
-        private CancellationToken CmdletCancellationToken;
-
```

Comment:
If these CancellationTokens are not required, then why we defined them earlier and now deleting?